packages/core/src/types.ts

```diff
@@ -28,6 +28,12 @@ export interface KeyPair {
   secretKey: CryptoKey
 }
 
+export { Doc } from 'automerge'
```

Comment:
I think the idea was to prevent consuming code from having to refer directly to Automerge. 

I've removed this for now since it's not being used. 
8a24f7ded2e24ee75c256384840a140223d930e8